### PR TITLE
Show seams in piercing demo

### PR DESCRIPTION
**Pull request by jculvey**

### CODE REVIEW

**Comment by jculvey** [comment]
image

File: cloud-gallery/header/src/Slider/Slider.tsx [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+  component$,
+  useStore,
+  useStylesScoped$,
+  $,
+  useClientEffect$,
+  useRef,
+} from "@builder.io/qwik";
 import { getCookie, saveCookie } from "helpers";
-import styles from "./Slider.css?inline";
+import styles from "./Slider.scss?inline";
```
Comment (dario-piotrowicz) [inline cloud-gallery/header/src/Slider/Slider.tsx:10]: scss??? is that supported by qwik? 😮

File: cloud-gallery/header/src/Slider/Slider.scss [hunk 2]
```diff
@@ -26,3 +27,13 @@
...
+
+.seam-options-container {
+	.label {
+		margin-left: .5rem;
+	}
+
+	input {
+		margin-right: .5rem;
+	}
+}
```
Comment (dario-piotrowicz) [inline cloud-gallery/header/src/Slider/Slider.scss:39]:
```suggestion
}

```

**Comment by dario-piotrowicz** [comment]
Could the seams be of different colors depending on the fragment? 🙂

Also in the piercing demo the root one shouldn't say "fragment" but something along the lines or "legacy app" since that is not actually a fragment

Also I have a feeling that the text is slightly too big, I am wondering if we can make it a bit smaller and unobtrusive (maybe without a background? 🤔)

(PS: if you want to move to something else I can make the changes in the piercing demo 🙂)

File: cloud-gallery/header/src/Slider/Slider.tsx (hunk 1 above)
Comment (jculvey) [reply cloud-gallery/header/src/Slider/Slider.tsx:10]: Indeed it is :)

File: migration-strategies/piercing/app/productivity-suite/index.html [hunk 3]
```diff
@@ -96,6 +96,34 @@
         border-left: 1px solid currentColor;
         padding-left: 0.5rem;
       }
+
+      body.show-seams #root,
```
Comment (petebacondarwin) [inline migration-strategies/piercing/app/productivity-suite/index.html:100]: Ah! That's how you do it! Nice TIL.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
We should not delete `package-lock.json` it is essential for ensuring that we can get the same result for a particular commit for ever.

File: cloud-gallery/header/src/Slider/Slider.tsx (hunk 1 above)
Comment (dario-piotrowicz) [reply cloud-gallery/header/src/Slider/Slider.tsx:10]: 😮 👍

**Comment by jculvey** [comment]
Ok I added back the package-lock.json and tweaked the seam styles. Here's how it's looking now:

Screen Shot 2022-10-21 at 2 04 05 PM

Screen Shot 2022-10-21 at 2 01 56 PM

**Comment by dario-piotrowicz** [comment]
Awesome! thanks @jculvey 😍

**Pull request merged**